docs/example/accordionDocs.vue

```diff
@@ -39,13 +39,13 @@
     <h4>Panel has many types that change its appearance.</h4>
     <div class="bs-example">
       <h5><code>Primary</code> type</h5>
-      <panel header="Primary - Click to expand" type="primary">
+      <panel header="## Primary - Click to expand" type="primary">
```

Comment:
- `####` (h4) is large enough without being jarring.
- Add a `<br>` between these two panels, since each example (h4) is larger than its heading (h5).
- Update `doc-code` a few lines below. No need to show `<br>` tag.

Reply:
You misinterpreted the 3rd point.

Update the `doc-code` lines: [R56](https://github.com/MarkBind/vue-strap/pull/15/commits/3d82cf990be3a78ef639a5e75aca46706856aaa4#diff-f3fc088ed87b750b3bc64c0a37593646R56), [R59](https://github.com/MarkBind/vue-strap/pull/15/commits/3d82cf990be3a78ef639a5e75aca46706856aaa4#diff-f3fc088ed87b750b3bc64c0a37593646R59)
Revert removal of `<br>` tag: [L62](https://github.com/MarkBind/vue-strap/pull/15/commits/3d82cf990be3a78ef639a5e75aca46706856aaa4#diff-f3fc088ed87b750b3bc64c0a37593646L62)